docs/remote-bazel-introduction.md

```diff
@@ -0,0 +1,228 @@
+---
+id: remote-bazel-introduction
+title: Introduction to Remote Bazel
+sidebar_label: Remote bazel introduction
+---
+
+Remote Bazel is an easily configurable way to run commands on a remote runner. You
+can think of it as dynamically spinning up a VM to execute a single command.
+This means you don't even need Bazel installed
+on your local machine to initiate a Bazel build! Plus, our remote
+runners support any bash commands, not just Bazel commands.
+
+## Benefits of Remote Bazel
+
+Remote Bazel makes it easy to configure the OS, architecture, and container image
+of the remote runner. This makes it easy to run builds that must run on a specific
+platform.
+
+Remote Bazel also has the following performance benefits:
+
+1. Colocation with BuildBuddy servers, ensuring a **fast network
+   connection between Bazel and BuildBuddy's RBE & caching servers**.
+2. Bazel workspaces are recycled between runs, allowing subsequent runs to
+   take advantage of **warm Bazel instances**.
+
+Remote Bazel uses the same backend architecture as our CI product, BuildBuddy
+Workflows. See [our docs on BuildBuddy Workflows](https://www.buildbuddy.io/docs/workflows-introduction/)
+for a more in-depth description of the performance optimizations and mechanism
+for workspace recycling.
+
+## Uses for Remote Bazel
+
+### CI
+
+Remote Bazel is powered by the same backend as our
+CI product, BuildBuddy Workflows. However, Remote Bazel is a much more flexible
+entrypoint to our CI platform.
+
+Workflows must be configured with a config YAML that is checked in to GitHub.
+Despite the performance benefits of running on Workflows, many companies have
+legacy CI workflows that would be challenging and error-prone
+to migrate to a new CI platform.
+
+Remote Bazel, on the other hand, is intended as a drop-in solution that should be
+easy to integrate into pre-existing CI pipelines. It can
```

Comment:
Remote Bazel is a drop-in solution that can be more easily integrated into pre-existing CI pipelines.

Reply:
Much better. Thanks!